src/SmartComponents/SystemInventory/SystemInventoryCard.js

```diff
@@ -8,31 +14,97 @@ import messages from '../../Messages';
 /**
  * System inventory card for showing system inventory and status.
  */
-const SystemInventoryCard = () => {
+const SystemInventoryCard = ({
+    fetchInventory, inventoryFetchStatus, inventorySummary,
+    fetchInventoryStale, inventoryStaleFetchStatus, inventoryStaleSummary,
+    fetchInventoryWarning, inventoryWarningFetchStatus, inventoryWarningSummary,
+    fetchInventoryTotal, inventoryTotalFetchStatus, inventoryTotalSummary
+}) => {
+
+    useEffect(() => {
+        fetchInventoryTotal();
+        fetchInventory();
+        fetchInventoryStale();
+        fetchInventoryWarning();
+    }, [fetchInventoryTotal,
+        fetchInventory,
+        fetchInventoryStale,
+        fetchInventoryWarning]
+    );
 
     const intl = useIntl();
 
     return <TemplateCard appName='SystemInventory'>
         <TemplateCardHeader subtitle={ intl.formatMessage(messages.systemInventoryTitle) }/>
         <TemplateCardBody isFilled={ false }>
-            <NumberDescription
-                data="100000"
-                dataSize="lg"
-                percentageData={ intl.formatMessage(messages.systemInventoryPercentageData) }
-                linkDescription={ intl.formatMessage(messages.systemInventoryDescription) }
-            />
-            <IconInline
-                message={ intl.formatMessage(messages.systemInventoryWarning) }
-                state="warning"
-                systemInventory="true"
-            />
-            <IconInline
-                message={ intl.formatMessage(messages.systemInventoryDanger) }
-                state="critical"
-                systemInventory="true"
-            />
+            { inventoryFetchStatus === 'fulfilled' && inventoryTotalFetchStatus === 'fulfilled' &&
+                <NumberDescription
+                    data={ inventorySummary.total || 0 }
```

Comment:
does this number get a comma after every three digits https://marvelapp.com/a5c8bcg/screen/66441550

Reply:
Ah good catch!